Formula/luarocks.rb

```diff
@@ -4,29 +4,45 @@ class Luarocks < Formula
   sha256 "0e1ec34583e1b265e0fbafb64c8bd348705ad403fe85967fd05d3a659f74d2e5"
   head "https://github.com/keplerproject/luarocks.git"
 
+  patch :DATA
+
   depends_on "mashape/kong/openresty"
 
   def install
     openresty = Formula["mashape/kong/openresty"]
 
-    # Install to the Cellar, but the tree to install modules is in HOMEBREW_PREFIX
+    # Install to the Cellar, but the tree to install modules is in HOMEBREW_PREFIX/opt/kong
     args = [
       "--prefix=#{prefix}",
-      "--rocks-tree=#{openresty.prefix}/luajit",
+      "--rocks-tree=#{HOMEBREW_PREFIX}/opt/kong",
       "--sysconfdir=#{prefix}/etc",
       "--with-lua=#{openresty.prefix}/luajit",
       "--with-lua-include=#{openresty.prefix}/luajit/include/luajit-2.1",
       "--lua-suffix=jit"
     ]
 
+    ENV.deparallelize
```

Comment:
This disables the `-j4` option in `make`. I believe this is not strictly necessary right now, but when I tried `make bootstrap`, the build failed badly because the LuaRocks makefile is not parallel-friendly. I ended up switching back to `make build`, but since at this point I can't vouch for its parallel-correctness, I thought it would be best to stay on the safe side and keep it here, until I actually check for and fix the implicit dependencies in the LuaRocks makefile. The LuaRocks `make` process is super-fast anyway, so having `-j` there or not is not a big deal.